#!/usr/bin/env python
PACKAGE = "ouster_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_range", double_t, 0, "min range", 1.0, 0.5, 120.0)
gen.add("max_range", double_t, 0, "max range", 120.0, 0.5, 120.0)
gen.add("image_width", int_t, 0, "image width", 1024, 16, 2048)
gen.add("organized", bool_t, 0, "return organized cloud", True)

exit(gen.generate(PACKAGE, PACKAGE, "OusterOS1"))
